http/server_sent_event_stream.ts

```diff
@@ -49,7 +59,11 @@ function stringify(message: ServerSentEventMessage): Uint8Array {
     );
   }
   if (message.id) {
-    assertHasNoNewline(message.id.toString(), "`message.id`");
+    assertHasNoNewline(
+      message.id.toString(),
+      "`message.id`",
+      "Cannot stringif messagey",
```

Comment:
I prefer 'serialize' verb here as this function returns Uint8Array instead of string, though it's named 'stringify'..

```suggestion
      "Cannot serialize message",
```